spe1_precsalt/SPE1CASE1_PRECSALT.DATA

```diff
@@ -115,7 +115,7 @@ PROPS
 
 --Solubility limit lb/stb and solid salt density  lb/ft3
 SALTSOL
-	0.5 135.468674 /
+	0.5 135.468675025 /
```

Comment:
I get the (slightly) different value
```
135.4686744502338
```
when I convert 2170 kg/m^3 to lb/ft^3.  Can you check your value again, please?

Reply:
Ok, if use the conversion values as it's in the Units.hpp, I got this 135.46867445023383